store/sqlstore/upgrade.go

```diff
@@ -1425,7 +1425,9 @@ func upgradeDatabaseToVersion630(sqlStore *SqlStore) {
 	sqlStore.CreateColumnIfNotExists("Schemes", "DefaultPlaybookMemberRole", "VARCHAR(64)", "VARCHAR(64)", "")
 	sqlStore.CreateColumnIfNotExists("Schemes", "DefaultRunAdminRole", "VARCHAR(64)", "VARCHAR(64)", "")
 	sqlStore.CreateColumnIfNotExists("Schemes", "DefaultRunMemberRole", "VARCHAR(64)", "VARCHAR(64)", "")
-
+	sqlStore.RemoveIndexIfExists("idx_uploadsessions_user_id", "UploadSessions")
```

Comment:
@streamer45 I think we should check if the index is actually on the incorrect column to avoid deleting/creating indexes on every restart. WDYT?